maxim/logger/groq/helpers.py

```
@@ -0,0 +1,89 @@
+from ..logger import Generation, Trace
+from ...scribe import scribe
+
+class GroqHelpers:
+    """Helper functions for Groq SDK."""
+
+    @staticmethod
+    def sync_stream_helper(
+        stream: Stream[ChatCompletionChunk],
+        generation: Generation,
+        trace: Optional[Trace],
+        is_local_trace: bool,
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)